src/main/scala/ch/epfl/bluebrain/nexus/kg/marshallers/instances/package.scala

```diff
@@ -40,8 +40,20 @@ package object instances extends FailFastCirceSupport {
   final implicit def httpEntity[A](implicit encoder: Encoder[A],
                                    printer: Printer = Printer.noSpaces.copy(dropNullValues = true),
                                    keys: OrderedKeys = OrderedKeys(
-                                     List("@context", "@id", "@type", "code", "message", "details", "")))
-    : ToEntityMarshaller[A] =
+                                     List("@context",
```

Comment:
I guess we could add the default behaviour of the orderedKeys directly in `AppConfig` as implicit val, since it can be reused anywhere we need it